heartex/label-studio/example-values/label-studio-on-openshift.yaml

```diff
@@ -0,0 +1,31 @@
+# Label Studio running on OpenShift
+
+app:
+  podSecurityContext:
+    enabled: false
+  containerSecurityContext:
+    enabled: false
+  # ingress:
+  #   enabled: true
+
+rqworker:
+  podSecurityContext:
+    enabled: false
+  containerSecurityContext:
+    enabled: false
+
+postgresql:
+  enabled: true
+  primary:
+    podSecurityContext:
+      enabled: false
+    containerSecurityContext:
+      enabled: false
+
+redis:
```

Comment:
rqworker is not required for LabelStudio OpenSource

Reply:
If someone is installing LabelStudio enterprise could they use rqworker? The example is not exclusive.

Reply:
Have updated the notes in the values

Reply:
I suggest to split this example into two: `label-studio-on-openshift.yaml` and `label-studio-enterprise-on-openshift.yaml`

Reply:
@farioas 

I do not have a license to test the enterprise version with. I am removing the comments and will only focus on a community deployment